src/common/Constants.ts

```diff
@@ -1,4 +1,20 @@
-import { CHAIN_IDs, TOKEN_SYMBOLS_MAP, ethers } from "../utils";
+import { CHAIN_IDs, TOKEN_SYMBOLS_MAP, ethers, Signer, Provider } from "../utils";
+import {
+  BaseBridgeAdapter,
+  DefaultERC20Bridge,
```

Comment:
Might be nice to scope this import so that it's easy to spot what type of chain it is.

```suggestion
  DefaultERC20Bridge as OVMDefaultERC20Bridge,
```

Reply:
Fixed. I specified both this and the Weth bridge to be OpStack[bridge]: https://github.com/across-protocol/relayer/pull/1524/commits/5aacd351220a191e4ba57c07e729bccbbdb36020